third_party/zlib/contrib/gcc_gvmat64/gvmat64.S

```diff
@@ -1,12 +1,12 @@
 /*
 ;uInt longest_match_x64(
 ;    deflate_state *s,
-;    IPos cur_match);                             // current match 
+;    IPos cur_match);                             // current match
```

Comment:
Please also revert this thirds party file.

Reply:
Reverted all third_party files.